doc/articles/toc.yml

```diff
@@ -40,6 +40,13 @@
           href: guides\native-frame-nav-tutorial.md
         - name: How to consume a web API
           href: howto-consume-webservices.md
+        - name: How to use Uno.Material
+          topicHref: guides/uno-material-walkthrough.md
+          items:
+          - name: Overview
+            href: guides/uno-material-walkthrough.md
+          - name: Controls extra setup
```

Comment:
Don't nest it here, nest it under the feature article (`features/uno-material.md`)